jerry-core/vm/vm.h

```diff
@@ -213,6 +213,11 @@ typedef enum
   VM_OC_CONTEXT_END,             /**< context end */
   VM_OC_JUMP_AND_EXIT_CONTEXT,   /**< jump and exit context */
 
+  VM_OC_CREATE_BINDING,          /**< create variables */
+  VM_OC_SET_BYTECODE_PTR,        /**< setting bytecode pointer */
+  VM_OC_VAR_FUNC_EVAL,           /**< variable and function evaluation */
+  VM_OC_INIT_LOCAL,              /**< initialize local variable */
```

Comment:
Imo, this group opcode is noly used in es2015 profile. Please check/guard it accroding to it.

Reply:
I think some of them is used in ES5.1. But it is worth to check them.